### PR TITLE
Fix global types exports issue

### PR DESCRIPTION
**Pull request by kamiazya**
Disable rollupTypes in packages/react/vite.config.ts to fix global types exports issue.

## Summary by CodeRabbit

- **Chores**
  - Updated internal build and configuration files for improved compatibility and workflow management.

### CODE REVIEW

File: .github/workflows/pr-snapshot-release.yaml [hunk 1]
```diff
@@ -106,8 +106,6 @@ jobs:
         with:
           node-version: 20
```
Comment (nagasawaryoya) [inline .github/workflows/pr-snapshot-release.yaml:107]: As an aside, I think you can always use the node version of your project by using the `node-version-file`.
https://github.com/actions/setup-node?tab=readme-ov-file#usage

**Review by nagasawaryoya — COMMENTED** [review]
@kamiazya 
Everything else seems fine!

File: .github/workflows/pr-snapshot-release.yaml (hunk 1 above)
Comment (kamiazya) [reply .github/workflows/pr-snapshot-release.yaml]: Thanks! Fixed it at b1e4bc7.

**Review by nagasawaryoya — APPROVED** [review]
LGTM!

**Pull request merged**